e2e/provision/README.md

```diff
@@ -118,14 +123,192 @@ connection with that setting).
    from the cluster. For example:
 
    ```
+   $ kubectl get clusters
+   NAME       PHASE         AGE     VERSION
+   edge01     Provisioned   3h40m   v1.26.3
+   edge02     Provisioned   3h42m   v1.26.3
+   edge03     Provisioned   3h42m   v1.26.3
+   edge04     Provisioned   3h40m   v1.26.3
    $ kubectl get secret edge01-kubeconfig -o jsonpath='{.data.value}' | base64 -d > edge01-kubeconfig
    $ kubectl --kubeconfig edge01-kubeconfig get ns
    NAME                           STATUS   AGE
-   config-management-monitoring   Active   5m25s
-   config-management-system       Active   5m25s
-   default                        Active   5m46s
-   kube-node-lease                Active   5m48s
-   kube-public                    Active   5m48s
-   kube-system                    Active   5m48s
+   config-management-monitoring   Active   3h35m
+   config-management-system       Active   3h35m
+   default                        Active   3h39m
+   kube-node-lease                Active   3h39m
+   kube-public                    Active   3h39m
+   kube-system                    Active   3h39m
    $
    ```
+
+   You should also check that the Kind cluster came up fully with `kubectl get
+   machinesets`. Sometimes they do not all come up; it's not clear why yet, but
+   is probably a resourcing issue on the VM. For example, in this case only
+   two clusters had machines come up:
+
+   ```
+   $ kubectl get machinesets
```

Comment:
Is this more of a troubleshooting step or verification step?

Reply:
I guess a verification step. We want to make sure that the kind clusters have come up.